### DISCUSSION TITLE
Help, I tried to create bot with microsoft auth!

### DISCUSSION THREAD

**Discussion by Skublik-glek** [PrismarineJS/mineflayer#2305] [Q&A]
Help, I tried to create bot with microsoft auth! I have 2FA. Is it a problem?
bot = mineflayer.createBot({
      "username": 'my email',
      "password": 'my password',
      "host": 'mc.hypixel.net',
      "port": 25565,
      "version": "1.8.9",
      "authTitle": '00000000402b5328',
      "auth": 'microsoft'                  
})

**Comment by U5B** [comment]
remove password and follow instructions given.
